src/env.cc

```diff
@@ -341,6 +344,18 @@ void Environment::ToggleImmediateRef(bool ref) {
 }
 
 
+Local<Value> Environment::GetNow() {
+  uv_update_time(event_loop());
+  uint64_t now = uv_now(event_loop());
+  CHECK(now >= timer_base());
+  now -= timer_base();
+  if (now <= 0xfffffff)
```

Comment:
I think you could go until at least `0x7fffffff` here?

Reply:
I think that might actually be missing an extra `f`? I think it's supposed to be `2**32 - 1`, no?

Reply:
@apapirovski If you want that then yes, one more `f` than in the current code it is (`0xffffffff`)

Reply:
Yeah, looks like that's been incorrect since it got first committed like 3-4 years ago :)

Reply:
Thanks! :)